src/Serilog.Sinks.MSSqlServer/project.json

```diff
@@ -1,22 +1,33 @@
 {
-  "version": "4.0.0-beta-0",
-  "description": "A Serilog sink that writes events to Microsoft SQL Server",
+  "version": "4.0.0-rc-*",
+  "description": "A Serilog sink that writes events to Microsoft SQL Server.",
   "authors": [ "Serilog Contributors" ],
-  "tags": [ "serilog", "sinks", "mssqlserver" ],
-  "projectUrl": "http://serilog.net",
-  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
-  "iconUrl": "http://serilog.net/images/serilog-sink-nuget.png",
+
   "dependencies": {
-    "Serilog": "2.0.0-beta-523",
-    "Serilog.Sinks.PeriodicBatching": "2.0.0-beta-700"
-  },
-  "compilationOptions": {
-    "keyFile": "../../assets/Serilog.snk"
+    "Microsoft.Extensions.Configuration": "1.0.0",
+    "Serilog": "2.0.0",
+    "Serilog.Sinks.PeriodicBatching": "2.0.0"
   },
+
   "frameworks": {
-    "net45": {}
+    "net452": {
```

Comment:
Any reason not to target `net4.5` here? 2.0 drops .NET 4 support but 4.5 is still supported wherever we can.